code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -461,7 +461,13 @@
 		H.see_in_dark = 8
 		if(!H.druggy)
 			H.see_invisible = SEE_INVISIBLE_LEVEL_TWO
+		if(H.client)
+			H.overlay_fullscreen("pmaster_lighting_g", /obj/screen/fullscreen/pmaster_g_default, 1)
```

Comment:
It really doesn't help when you just say "no" and nothing else. I'm not sure exactly what you are expressing disapproval to. Is it the way I called this function, is it where it is, or what?